lib/api/policy.go

```diff
@@ -118,6 +118,9 @@ type PolicySpec struct {
 	// PreDNAT indicates to apply the rules in this policy before any DNAT.
 	PreDNAT bool `json:"preDNAT,omitempty"`
 
+	// ApplyOnForward indicates to apply the rules in this policy on forward traffic.
+	ApplyOnForward bool `json:"applyonforward,omitempty"`
```

Comment:
I think the the JSON name should be `applyOnForward` to match the casing we use elsewhere.